tmf/org.eclipse.tracecompass.tmf.core/src/org/eclipse/tracecompass/tmf/core/model/StyleProperties.java

```diff
@@ -283,7 +283,10 @@ public interface FontWeight {
     /**
      * Symbol type as a string. Possible values: {@link SymbolType}.
      * <p>
-     * Default: @link {@link SymbolType#NONE}
+     * Default: {@link SymbolType#NONE}, which is {@link SymbolType#DIAMOND}.
```

Comment:
Revert this change, since it does not actually draw a diamond for NONE.

Reply:
done